lib/ember_cli.rb

```diff
@@ -6,36 +6,7 @@ def self.dist_dir
   end
 
   def self.assets
-    @assets ||=
-      begin
-        assets = %w[
-          discourse.js
-          admin.js
-          wizard.js
-          ember_jquery.js
-          markdown-it-bundle.js
-          start-discourse.js
-          vendor.js
-        ]
-        assets +=
-          Dir.glob("app/assets/javascripts/discourse/scripts/*.js").map { |f| File.basename(f) }
-
-        if workbox_dir_name
-          assets +=
-            Dir
-              .glob("app/assets/javascripts/discourse/dist/assets/#{workbox_dir_name}/*")
-              .map { |f| "#{workbox_dir_name}/#{File.basename(f)}" }
-        end
-
-        Discourse
-          .find_plugin_js_assets(include_disabled: true)
-          .each do |file|
-            next if file.ends_with?("_extra") # these are still handled by sprockets
-            assets << "#{file}.js"
-          end
-
-        assets
-      end
+    @@assets ||= Dir.glob("**/*.{js,map,txt}", base: "#{dist_dir}/assets")
```

Comment:
What txt files are pulled here?

Reply:
Webpack extracts license information from our dependencies and sticks them in a txt file (and then puts a link to the txt file in the js file)